transcom/mymove
# B 20068 display weight

## [B-20068](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20068)

[Integration Pull Request](https://github.com/transcom/mymove/pull/13123)

## Summary
This story is a simple alteration to change the "Estimated Weight" value for NTSR shipments only to instead display it's data as the ntsRecordedWeight value of an NTSR Shipment on the PrimeUI. It does not impact any editing functions on the prime UI, only displaying the estimated weight as the previously recorded NTS weight, which is required to be entered by a TOO if not a Service Counselor before the Prime can see the shipment.

Please ensure client tests are passing, and other types of shipments still show the "Estimated Weight" as the actual estimated weight value.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. This is best tested with an NTSR set up on a new move. Log in as a customer, create a new move.
2. Create two shipments, both NTSR.
3. Once the shipments are submitted, log in as a Service Counselor. 
4. On one of the shipments, ensure you enter the previously recorded NTS weight. On the other shipment, do NOT enter the previously recorded NTS weight.
5. Submit the shipments.
6. Log in as a TOO.
7. Ensure that the previously recorded NTS weight is present for the shipment you already added it to. 
8. Ensure that the previously recorded NTS weight is required for the other shipment. Enter an amount.
9. Submit the moves.
10. Log in as a Prime, locate the move you've been using.
11. On the shipment details page, ensure both Estimated Weight values match those you had entered in the steps above.
<img width="966" alt="Screenshot 2024-06-21 at 2 44 24 PM" src="https://github.com/transcom/mymove/assets/148906585/bd0c6159-e470-4104-8cda-de21de76bec0">
12. View other shipment types as a Prime, and ensure their Estimated Weights match the estimated weight data in your network console/database.
13. Ensure client tests passing.

### Frontend
